.github/images/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM kylemanna/bitcoind:latest AS builder
+
+RUN apt-get update \
+    && apt-get install -y wget gnupg2 curl git
+
+# -- nodeJs ---------------------------------------------------------
+ENV NODE_VERSION  v20.14.0
+RUN mkdir -p /usr/local/nvm
+ENV NVM_DIR /usr/local/nvm
+
+RUN  curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash \
+     && . $NVM_DIR/nvm.sh \
+     &&  nvm install $NODE_VERSION \
+    && nvm alias default $NODE_VERSION \
+    && nvm use default
+
+ENV NODE_PATH $NVM_DIR/$NODE_VERSION/lib/node_modules
+ENV PATH $NVM_DIR/versions/node/$NODE_VERSION/bin:$PATH
+
+# -- java ---------------------------------------------------------
+ENV JAVA_VERSION 11
+
+RUN apt-get  -y install "openjdk-$JAVA_VERSION-jdk"
+
+ENV JAVA_HOME=/opt/java/openjdk \
+    PATH="/opt/java/openjdk/bin:$PATH"
+
+# -- python ---------------------------------------------------------
+ENV PYTHON_VERSION=2.7.16
```

Comment:
Are we going to use HSM1 or HSM2 here? For the first phase, we could simply ignore any HSM related stuff and use all key files. What do you think?

Reply:
I understand this, but I thought that it wouldn't hurt have it here. Since we had the effort to add support on the image, and that maybe we could use this image to build and run full integration tests with Python, what do you think?

Reply:
No need for them now. I see you removed it. 👍🏽